public/css/style.css

```diff
@@ -1343,11 +1343,11 @@
 
  body {
    background: #002266;
+   height: 100% !important;
  }
 
  .navigationDiv {
```

Comment:
There is no universal convention to naming css ids and classes, but pick anyone you like and stick to it. In this file only, there is camel case e.g .navigationDiv on line 1349, snake case e.g .abandon_game_button on line 1348 and hyphen separation e.g .share-button-app on line 1394